src/opm/parser/eclipse/EclipseState/Aquifetp.cpp

```diff
@@ -0,0 +1,57 @@
+/*
+  Copyright (C) 2017 TNO
+
+  This file is part of the Open Porous Media project (OPM).
+
+  OPM is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  OPM is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <opm/parser/eclipse/EclipseState/Aquifetp.hpp>
+
+
+namespace Opm{
+	
+  Aquifetp::Aquifetp(const EclipseState& eclState, const Deck& deck)
+  	{	
+      if (!deck.hasKeyword("AQUFETP"))
+        return;
+
+      const auto& aquifetpKeyword = deck.getKeyword("AQUFETP");
+
+      for (auto& aquifetpRecord : aquifetpKeyword){
+
+          Aquifetp::AQUFETP_data data;
+
+          data.aquiferID = aquifetpRecord.getItem("AQUIFER_ID").template get<int>(0);
+          data.pvttableID = aquifetpRecord.getItem("TABLE_NUM_WATER_PRESS").template get<int>(0);
+          data.p0 = aquifetpRecord.getItem("P0").getSIDouble(0);
+          data.d0 = aquifetpRecord.getItem("DAT_DEPTH").getSIDouble(0);
+          data.C_t = aquifetpRecord.getItem("C_T").getSIDouble(0);
+          data.V0 = aquifetpRecord.getItem("V0").getSIDouble(0);
+          data.J = aquifetpRecord.getItem("PI").getSIDouble(0);
+
+          m_aqufetp.push_back(std::move(data));
+      	}
+  	}
+
+  	const std::vector<Aquifetp::AQUFETP_data>& Aquifetp::getAquifers() const
```

Comment:
Instead of this method I would prefer if you exposed `begin()` and `end()` methods were forwarded to the underlying `std::vector`.